src/betting/wallet_impls/spend_won.rs

```diff
@@ -62,7 +62,7 @@ impl GunWallet {
             .filter_map(|result| match result {
                 Ok(ok) => Some(ok),
                 Err(e) => {
-                    eprintln!("Eror with entry in database: {}", e);
+                    elog!(@explosion "Error with entry in database: {}", e);
```

Comment:
I think in this case we are talking about an error which means "something is terribly wrong but we're ignoring it and continuing". I think this should be `@recoverable_error`. Maybe something more like :cold_sweat: for the emoji?

Reply:
On second thought explosion emoji might actually be more appropriate since you want to draw attention to it.

Reply:
yea I was thinking what would be more appropriate in this case and I wasn't sure. So should I switch something?

Reply:
The emoji is fine I think just change the `@` to `@recoverable_error`.